feat: Adds lockers count to the tvl endpoint

gluneau opened the pull request:
**Prerequisite:** graphql indexes have to be on v2 first 

Add 3 new enpoints with 3 values each for stakers and lockers. (date, count, amount)
and a combined endpoint.

`http://127.0.0.1:5001/astar-token-api/us-central1/app/api/v3/shiden/dapps-staking/stakers-total/90%20days`

```json
[
  {
    "date": "1708041600000",
    "count": 403,
    "amount": "1977589088000000000000000"
  },
  {
    "date": "1707955200000",
    "count": 397,
    "amount": "1967662208000000000000000"
  }, ...
```


`http://127.0.0.1:5001/astar-token-api/us-central1/app/api/v3/shiden/dapps-staking/lockers-total/90%20days`

```json
[
  {
    "date": "1708041600000",
    "count": 3343,
    "amount": "16286708604872965460014209"
  },
  {
    "date": "1707955200000",
    "count": 3344,
    "amount": "16286474924238392891442274"
  }, ...
```


`http://127.0.0.1:5001/astar-token-api/us-central1/app/api/v3/shiden/dapps-staking/lockers-and-stakers-total/90%20days`

```json
[
  {
    "date": "1708041600000",
    "tvl": "16286708604872965460014209",
    "lockersCount": 3343,
    "tvs": "1977589088000000000000000",
    "stakersCount": 403
  },
  {
    "date": "1707955200000",
    "tvl": "16286474924238392891442274",
    "lockersCount": 3344,
    "tvs": "1967662208000000000000000",
    "stakersCount": 397
  }, ...
```

gluneau committed:
adds lockers count to the tvl endpoint

bobo-k2 commented:
Maybe to return objects array from the call. When you have just timestamp and value (e.g.) tvl this is fine to have array, because it is more or less obvious from the endpoint name and data.

This change will probably break some client code. Think about introducing a new endpoint 

Return something like
```
[
  {
    timestamp: "1705363200000",
    tvl: "5475067296262039963124623",
    uniqueLockerAccounts: 21016
  }
]
```

gluneau committed:
3 new endpoints

gluneau commented:
> Maybe to return objects array from the call. When you have just timestamp and value (e.g.) tvl this is fine to have array, because it is more or less obvious from the endpoint name and data.
> 
> This change will probably break some client code. Think about introducing a new endpoint
> 
> Return something like
> 
> ```
> [
>   {
>     timestamp: "1705363200000",
>     tvl: "5475067296262039963124623",
>     uniqueLockerAccounts: 21016
>   }
> ]
> ```

I have reverted the endpoint changes and introduced 3 new endpoints.

bobo-k2 reviewed (approved):
LGTM, I think the combined lockers and stakers endpoint is not necessary, but it is ok to have it.
